src/dmd/expression.d

```diff
@@ -6188,7 +6252,7 @@ extern (C++) class CatAssignExp : BinAssignExp
     }
 }
 
-///
+/// ditto
```

Comment:
Maybe it would be better to split the previous documentation into smaller piece which are located next to the class? That way it would be consistent with the `BinAssignExp` documentation.